expression/builtin_arithmetic_vec.go

```diff
@@ -210,11 +211,124 @@ func (b *builtinArithmeticIntDivideIntSig) vecEvalInt(input *chunk.Chunk, result
 }
 
 func (b *builtinArithmeticPlusIntSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinArithmeticPlusIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	lh, err := b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(lh)
+
+	rh, err := b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(rh)
+
+	if err := b.args[0].VecEvalInt(b.ctx, input, lh); err != nil {
+		return err
+	}
+
+	if err := b.args[1].VecEvalInt(b.ctx, input, rh); err != nil {
+		return err
+	}
+
+	result.ResizeInt64(n, false)
+	result.MergeNulls(lh)
+	result.MergeNulls(rh)
```

Comment:
We can merge them at the same time `result.MergeNulls(lh, rh)`.

Reply:
Thanks for your advise.